lib/src/test/java/com/thenewboston/api/bankapi/datasource/BankDataSourceTest.kt

```diff
@@ -171,31 +171,16 @@ class BankDataSourceTest {
             }
 
             @Test
-            fun `should return success with original invalid block identifier`() = runBlockingTest {
-                // given
-                val request = Mocks.postInvalidBlockRequest()
-
-                // when
-                val response = bankDataSource.sendInvalidBlock(request)
-
-                // then
-                check(response is Outcome.Success)
-                response.value.id shouldNot beEmpty()
-                response.value.blockIdentifier shouldBe request.message.blockIdentifier
-            }
-
-            @Test
-            fun `should return success with balanceKey `() = runBlockingTest {
-                // given
-                val request = Mocks.postBlockRequest()
+            fun `should send validator confirmation successfully`() = runBlockingTest {
+                val request = Mocks.confirmationServiceRequest()
 
                 // when
-                val response = bankDataSource.sendBlock(request)
+                val response = bankDataSource.sendValidatorConfirmationServices(request)
 
                 // then
                 check(response is Outcome.Success)
-                response.value.id shouldNot beEmpty()
-                response.value.balanceKey shouldBe request.message.balanceKey
+                response.value.start shouldBe "2020-11-19T19:57:31.799872"
+                response.value.end shouldBe "2020-11-19T19:57:31.799872"
```

Comment:
Consider comparing against the values from the request. Otherwise, this test will fail after changing a mock value.

Reply:
Good point!